docs/sre-guide/support/community-domains.md

```diff
@@ -0,0 +1,173 @@
+
+## Preparing the configuration change
+
+1. In the `<hub-name>.values.yaml` file, change `jupyterhub.ingress.hosts` and
+   `jupyterhub.ingress.tls`, replacing the 2i2c domain name with the community
+   provided domain name.
+
+2. In `cluster.yaml` file for the cluster the hubs are
+   in, change the `domain` field to point to the community managed domain.
+
+3. If Auth0 is being used, change `jupyterhub.hub.config.CustomAuth0OAuthenticator.logout_redirect_to_url`
+   to point to the community managed domain.
+
+4. If GitHub Authentication is being used, change the domain used in the URL in
+   `jupyterhub.hub.config.GitHubOAuthenticator.oauth_callback_url` to point to the community
+   managed domain.
+
+5. If CILogon Authentication is being used, change the domain used in the URL in
+   `jupyterhub.hub.config.CILogonOAuthenticator.oauth_callback_url` to point to the community
+   managed domain.
+
+6. In the `support.values.yaml` file for the cluster in which the hubs are, we set up redirects
+   so folks who go to the older domains will get redirected to the new domain.
+
+   ```yaml
+   redirects:
+     rules:
+     - from: <2i2c-managed-domain>
+       to: <community-managed-domain>
+   ```
+
+7. Make a PR with your changes.
+
+## Deploying the configuration change
+
+If there is a staging hub in the list of hubs being changed, you can [deploy manually](hubs:manual-deploy)
+to test these changes. To test the redirects manually, you can use the [deploy-support](deploy-support-chart:manual)
+command.
+
+### Determine a possible deployment time window
+
+If this is a currently running hub with active users, this change will be *disruptive*, as the old
+URLs will no longer work. Communicate this to the community, and find a useful time to deploy
+the change that will be the least disruptive.
+
+However, most often this is done *before* there is any real usage on the hub, and timing this
+is not a consideration.
+
+### Authentication updates
+
+**Just** before deployment, we should change the Authorized Callback URLs in the appropriate
+upstream authentication provider we use. We delay this to just before the deployment, since in
+most providers we can have only *one* authorized callback URL. As soon as we change this URL to
+point to the new domain, users can not log in to the old domain anymore!
+
+#### GitHub Authentication
+
+1. Open the appropriate GitHub App from the [list of OAuth apps](https://github.com/organizations/2i2c-org/settings/applications)
+   in the 2i2c-org GitHub org.
+2. Change the **Homepage URL** to point to the new community managed domain.
+3. Change the **Authorization callback URL** to point to the new community managed domain. This
+   should match the value in `jupyterhub.hub.config.GitHubOAuthenticator.oauth_callback_url`.
+4. Hit the **Update application** button.
+
+#### CILogon Authentication
+
+```{todo}
+This section to be filled in :)
+```
```

Comment:
````suggestion

Use the `deployer cilogon-client` command to update the existing CILogon client application.

This commands needs to be passed the cluster name, the hub name and the **new domain**.

The example below updates the 2i2c's staging hub domain to ` new-domain.staging.2i2c.cloud`.

```bash
deployer cilogon-client update 2i2c staging new-domain.staging.2i2c.cloud
```

````{tip}
You can use the get command afterwards to verify that the update happened successfully and check the `redirect_urls` list in the output dictionary of the command.

```bash
deployer cilogon-client get 2i2c staging
```
`````